packages/ra-ui-materialui/src/Link.tsx

```diff
@@ -50,19 +50,19 @@ export interface LinkProps
 
 declare module '@mui/material/styles' {
     interface ComponentNameToClassKey {
-        RaLink: 'root' | 'link';
+        [PREFIX]: 'root' | 'link';
```

Comment:
Only root style is supported:
```suggestion
        [PREFIX]: 'root';
```